webapp/components/user_settings/user_settings_general.jsx

```diff
@@ -936,6 +963,99 @@ class UserSettingsGeneralTab extends React.Component {
             );
         }
 
+        let positionSection;
+        if (this.props.activeSection === 'position') {
+            let extraInfo;
+            let submit = null;
+            if ((this.props.user.auth_service === 'ldap' || this.props.user.auth_service === Constants.SAML_SERVICE) && global.window.mm_config.PositionAttributeSet === 'true') {
```

Comment:
If this is the case, we should add this to SAML and LDAP, if not we should remove this and add it later in case it applies